tests/playwright/sanity/container.test.js

```diff
@@ -2,45 +2,67 @@ const { test, expect } = require( '@playwright/test' );
 const { getElementSelector } = require( '../assets/elements-utils' );
 const WpAdminPage = require( '../pages/wp-admin-page' );
 
-test( 'Sort items in a Container using DnD', async ( { page }, testInfo ) => {
-	// Arrange.
-	const wpAdmin = new WpAdminPage( page, testInfo );
-	await wpAdmin.setExperiments( {
-		container: true,
-	} );
+test.describe( 'Container tests', () => {
+	test( 'Sort items in a Container using DnD', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Set row direction.
+		await page.click( '.elementor-control-flex_direction i.eicon-arrow-right' );
+
+		// Add widgets.
+		const button = await editor.addWidget( 'button', container ),
+			heading = await editor.addWidget( 'heading', container ),
+			image = await editor.addWidget( 'image', container );
 
-	const editor = await wpAdmin.useElementorCleanPost(),
-		container = await editor.addElement( { elType: 'container' }, 'document' );
+		// Act.
+		// Move the button to be last.
+		await editor.previewFrame.dragAndDrop(
+			getElementSelector( button ),
+			getElementSelector( image ),
+		);
 
-	// Set row direction.
-	await page.click( '.elementor-control-flex_direction i.eicon-arrow-right' );
+		// Wait for the button to re-render.
```

Comment:
Please see if you can replace the timeout with one of the page.waitfor functions 
https://playwright.dev/docs/api/class-page